app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_posts_counter
+    author.increment!(:postsCounter)
+  end
```

Comment:
- [Optional] COnsider adding `after_save` to call the `update_posts_counter` method every time a new like has been saved to the database, then make the update_psots_counter method private. This will save you the stress of manually updating posts counter every time a like is added.

```ruby
after_save :update_posts_counter
```